timesketch/api/v1/resources_test.py

```diff
@@ -43,6 +43,16 @@ def test_to_json_empty_list(self):
             },
         )
 
+class InvalidResourceTest(BaseTest):
+    """Test an Invalid Resource."""
+
+    invalid_resource_url = "api/v1/invalidresource"
+    
+    def test_invalid_endpoint(self):
```

Comment:
```suggestion
    invalid_resource_url = "api/v1/invalidresource"

    def test_invalid_endpoint(self):
```

Removing trailing whitespace making the linter happy.